welly/las.py

```diff
@@ -267,7 +267,7 @@ def datasets_to_las(path, datasets, **kwargs):
     for dataset_name, df in datasets.items():
 
         # dataset is the header
-        if dataset_name == 'Header':
+        if (dataset_name == 'Header'):
```

Comment:
Changes nothing in the code, just adds noise

```suggestion
        if dataset_name == 'Header':
```